github/resource_github_organization_ruleset.go

```diff
@@ -0,0 +1,575 @@
+package github
+
+import (
+	"context"
+	"log"
+	"net/http"
+	"strconv"
+
+	"github.com/google/go-github/v53/github"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/validation"
+)
+
+func resourceGithubOrganizationRuleset() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceGithubOrganizationRulesetCreate,
+		Read:   resourceGithubOrganizationRulesetRead,
+		Update: resourceGithubOrganizationRulesetUpdate,
+		Delete: resourceGithubOrganizationRulesetDelete,
+		Importer: &schema.ResourceImporter{
+			State: func(d *schema.ResourceData, meta interface{}) ([]*schema.ResourceData, error) {
+				return []*schema.ResourceData{d}, nil
+			},
+		},
+
+		SchemaVersion: 1,
+
+		Schema: map[string]*schema.Schema{
+			"name": {
+				Type:         schema.TypeString,
+				Required:     true,
+				ValidateFunc: validation.StringLenBetween(1, 100),
+				Description:  "The name of the ruleset.",
+			},
+			"target": {
+				Type:        schema.TypeString,
+				Required:    true,
+				Description: "Possible values are `branch` and `tag`.",
+			},
+			"repository": {
```

Comment:
Add `owner` field:
https://github.com/integrations/terraform-provider-github/pull/1808/files#r1272447174

Reply:
Added in https://github.com/o-sama/terraform-provider-github/pull/1